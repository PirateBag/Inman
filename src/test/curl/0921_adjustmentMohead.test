Try out some errors.
=== JSON ===
{ "rows" : [ { "amount" : "50", "itemId" : "2", "orderId" : "117", "orderType" : "MOHEAD", "effectiveDate" : "2025-0729", "adjustmentType" : "XFER", "crudAction" : "INSERT" }
 ] }