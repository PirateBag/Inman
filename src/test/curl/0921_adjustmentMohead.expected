{
  "responseType" : null,
  "errors" : [ {
    "key" : 0,
    "message" : "Adustment item ref (0) not found"
  } ],
  "data" : [ ]
}